kedro/framework/cli/cli.py

```diff
@@ -38,6 +39,9 @@
 v{version}
 """
 
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler(sys.stderr))
```

Comment:
With that setting, will every logging message be sent to `stderr`? If the `INFO` logging level is configured in Kedro, will `INFO` messages also go to `stderr`, or are they not connected to Kedro's logging settings?

Reply:
It should affect only the messages generated from this instance of the logger, which is just the one error message on the main function.